test/SparqlParserTest.cpp

```diff
@@ -1232,6 +1232,7 @@ TEST(ParserTest, Order) {
         pq._rootGraphPattern._children[0].variant_));
   }
   {
+    // Ordering by variables that are not grouped is not allowed.
     EXPECT_THROW(
         SparqlParser("SELECT ?x WHERE { ?x :myrel ?y } GROUP BY ?x ORDER BY "
                      "?y")
```

Comment:
I didn't see this test case in the last review, because it is not next to the other "failing" test cases. Please move it to the end, s.t. that the illegal inputs are next to each other.

Reply:
The line break is also unfortunate in this case.